docs/emulation.md

```diff
@@ -0,0 +1,32 @@
+
+# Emulation in Lighthouse
+
+In Lighthouse, "Emulation" refers to the screen/viewport emulation and UserAgent string spoofing.
+["Throttling"](./throttling.md) covers the similar topics around network and CPU throttling/simulation.
+
+With the default configuration, Lighthouse emulates a mobile device. There's [a `desktop` configuration](../lighthouse-core/config/desktop-config.js), available to CLI users with `--preset=desktop`, which applies a consistent desktop environment and scoring calibration. This is recommended as a replacement for `--emulation-form-factor=desktop`.
```

Comment:
```suggestion
With the default configuration, Lighthouse emulates a mobile device. There's [a `desktop` configuration](../lighthouse-core/config/desktop-config.js), available to CLI users with `--preset=desktop`, which applies a consistent desktop environment and scoring calibration. This is recommended as a replacement for `--form-factor=desktop`.
```

Or if this was referring to the v6 flags then `--emulated-form-factor`?

Reply:
ah yeah typo. i meant to refer to refer to that (i added this to docs excepting we'll get people who used -eff=desktop without problems before and want the quick answer)

thx